human-biomolecular-data/data-sources.md

```diff
@@ -19,93 +25,167 @@ training:
 
-## Introduction
+Human biomolecular data refers to information obtained from the analysis of biological molecules, such as DNA, RNA, proteins, and metabolites. This type of data is used to study the molecular mechanisms underlying disease and to identify potential drug targets.
+
+Human clinical data, on the other hand, refers to information obtained from the study of patients, including their medical histories, physical exams, and laboratory tests. This type of data is used to diagnose and treat disease, as well as to evaluate the safety and efficacy of new therapies.
 
-## What is human biomolecular data, and why is it important for infectious diseases research?
+While both types of data are important for understanding human health and disease, they are collected and analysed in different ways and for different purposes. 
 
-## Topic 1 
+Human biomolecular data is of great importance in infectious disease research because it plays a critical role in understanding the molecular basis of the disease. By analyzing the biomolecular data, researchers can gain a deeper understanding of the disease's pathogenesis, evolution, and transmission. Personalised medicine is another area where biomolecular data can be applied. By analyzing an individual's biomolecular data, researchers can develop personalised treatment plans that are tailored to the specific needs of the patient. For example, if a patient has a genetic mutation that predisposes them to a particular disease, this information can be used to develop a personalised treatment plan that takes into account the patient's genetic profile.
 
-<!--- Subsection related to a specific topic related to the data sources of the category that you selected.--->
+It is important to mention that most analyses of human biomolecular data are complemented by using the clinical data related to them.
 
-Short explanation of what this topic is about and why it is important, with an emphasis on infectious diseases and the category that you selected.
+<!-- ## Data deposition and FAIRness -->
+
+<!--- Currently in standby waiting to check the clinical data sources FAIRness.--->
+
+<!-- ### Considerations -->
+
+## Data deposition
+
+For advancing in the understanding of human biomolecular data, data deposition plays a pivotal role. The deposition of this data in publicly accessible databases serves as a valuable resource for scientists and researchers worldwide. It enables the integration and analysis of diverse datasets as well as by sharing and archiving data, researchers can ensure the reproducibility and transparency of their findings, allowing others to validate and build upon their work, encouraging collaboration. 
+In summary, data deposition not only fuels scientific progress but also empowers the global scientific community to unlock the complexities of human biology, ultimately leading to improved health outcomes and advancements in medical research.
```

Comment:
So far I had understood that the goal was to focus on finding data sources for secondary use on these data sources pages. However, I really like that you included a section on data deposition. Maybe this is something we could discuss during an upcoming meeting? I think that it would be good to chose one approach for all data sources pages.

Reply:
During T4.1 call on 8/8/2023 decided to work on this during virtual contentathon in September.

Reply:
we decided to add this in the template of the data sources pages as a "nice to have" topic, but not mandatory since might be that not everyone has the knowledge about this when writting the page.